src/bitmap.rs

```diff
@@ -130,3 +137,13 @@ impl<const PAGE_SIZE: usize> PageAllocator for BitmapPageAllocator<PAGE_SIZE> {
         self.total_pages - self.used_pages
     }
 }
+
+impl<const PAGE_SIZE: usize> BitmapPageAllocator<PAGE_SIZE> {
+    /// Checks whether the address has the demanded alignment.
+    ///
+    /// Equivalent to `addr % align == 0`, but the alignment must be a power of two.
+    #[inline]
+    const fn is_aligned(base_addr: usize) -> bool {
```

Comment:
No need to put it inside a impl block

Reply:
may we need the const type  `PAGE_SIZE`  associated with the `PageAllocator` ?